front_end/package-lock.json

```diff
@@ -21,6 +21,12 @@
         "sumchecker": "^3.0.1"
       }
     },
```

Comment:
We probably don't need to update this file as it gets updated automatically when we do "npm install".

Reply:
It seems that the package-lock is required for the tests to run properly.